assets/queries/openAPI/responses_wrong_http_status_code/query.rego

```diff
@@ -0,0 +1,35 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	response := doc.paths[n][oper].responses[code]
```

Comment:
I think we should check responses on [component object](https://swagger.io/specification/#responses-object)...

Reply:
I think responses in component object don't use http status code, they are used for ref, ex:
```
"responses": {
    "NotFound": {
      "description": "Entity not found."
    },
    "IllegalInput": {
      "description": "Illegal input for operation."
    },
    "GeneralError": {
      "description": "General Error",
      "content": {
        "application/json": {
          "schema": {
            "$ref": "#/components/schemas/GeneralError"
          }
        }
      }
    }
...
"get": {
    "responses": {
         "400": {
             "$ref": "#components/responses/NotFound"
         }
     }
}
```

Reply:
Make sense